src/Test/Perf/util/test_util.csx

```diff
@@ -200,6 +216,41 @@ ProcessResult ShellOut(
     };
 }
 
+ProcessResult ShellOutUsingShellExecute(
```

Comment:
I'm calling robocopy from ```ShellOut``` in ```fetch_build.csx```.  If needed, we should just create a CopyDirectory function that wraps that.  The code below troubles me for two reasons:

1.  We never check the exit code of the process
2.  We drop the stderr/stdout of the process on the floor.

If something goes wrong, it's going to be very difficult to diagnose it from a log of the script output.